### PR TITLE
Technote for new R package restez

### PR DESCRIPTION
**Pull request by DomBennett**
Technote for new R package [`restez`](https://github.com/ropensci/restez)

### CODE REVIEW

**Comment by stefaniebutland** [comment]
Thanks for submitting this @DomBennett! I will get you feedback this week,

File: content/technotes/2018-12-03-restez-release.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+```
+
+Alternatively, the latest development version can be downloaded from `restez`'s [GitHub page](https://github.com/ropensci/restez).
+
+```r
+devtools::install_github(repo = 'ropensci/restez')
+```
+
+## Usage
+
```
Comment (stefaniebutland) [inline content/technotes/2018-12-03-restez-release.md]: Please give a couple of sentences outlining the example you're going to walk through. Indicate what you might do with a set of phage sequences. i.e. can you turn it into an interesting example that might interest readers?

File: content/technotes/2018-12-03-restez-release.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+To minimise the coding effort on the part of a user, `restez` has been built to work with R packages that already connect to
+[NCBI's Entrez](https://www.ncbi.nlm.nih.gov/books/NBK49540/). After setting up a `restez` database the same functions of these
+other packages can be used to query NCBI Entrez. Internally, `restez` will query its local database and if it cannot find all
+of the requested sequences it will pass these arguments on to these other packages.
+
+For example, users can use the `entrez_fetch()` function of the [`rentrez`](https://github.com/ropensci/rentrez) package. Running
+this function through `restez` means a user can first check the local database rather than make lots of queries over the internet.
+The function arguments are exactly the same.
+Additionally, user's can set-up up a `restez` database before launching a [`phylotaR`](https://github.com/ropensci/phylotaR) run.
```
Comment (stefaniebutland) [inline content/technotes/2018-12-03-restez-release.md]: user's -> users

File: content/technotes/2018-12-03-restez-release.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+`phylotaR` searches NCBI for orthologous sequence clusters for a given taxonomic ID. If a `restez` database is set-up, `phylotaR`
```
Comment (stefaniebutland) [inline content/technotes/2018-12-03-restez-release.md]: set-up -> set up

File: content/technotes/2018-12-03-restez-release.md [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+
+* [The `restez` website](https://ropensci.github.io/restez/)
+* [`rentrez`](https://github.com/ropensci/rentrez#rentrez)
+* [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)
+* [`restez` on GitHub](https://github.com/ropensci/restez)
+* [`phylotaR`](https://github.com/ropensci/phylotaR)
+
+## Reference
+
+Bennett, D.J., Hettling, H., Silvestro, D., Vos, R. and Antonelli, A. **2018**. restez: Create and Query a Local Copy of GenBank in R.
```
Comment (stefaniebutland) [inline content/technotes/2018-12-03-restez-release.md]: Please update reference when published 🎉

File: content/technotes/2018-12-03-restez-release.md [hunk 5]
```diff
@@ -0,0 +1,336 @@
+---
+slug: restez
+title: 'restez: Locally query GenBank'
+package_version: 1.0.0
+authors:
+  - name: Dom Bennett
+    url: https://github.com/dombennett
+date: '2018-12-03'
+categories: technote
+topicid: 1292
```
Comment (stefaniebutland) [inline content/technotes/2018-12-03-restez-release.md]: topicid: 1292 -> topicid: 

I'll fill the topicid at publication time.

**Comment by DomBennett** [comment]
Hi @stefaniebutland,

Thanks for reading and editing! I made all the minor changes and I added a richer example to describe the basics of `restez` using Escherichia bacteriophages.

Is that any better?

Thanks!

**Comment by stefaniebutland** [comment]
Nice example addition thank you. 
Looks ready to publish. We'll tag you in a tweet about the post on Dec 3!

**Pull request merged**